core/src/test/scala/integration/kafka/api/BaseProducerSendTest.scala

```diff
@@ -43,6 +43,7 @@ abstract class BaseProducerSendTest extends KafkaServerTestHarness {
     val overridingProps = new Properties()
     val numServers = 2
     overridingProps.put(KafkaConfig.NumPartitionsProp, 4.toString)
+    overridingProps.put(KafkaConfig.OffsetsTopicPartitionsProp, "5") //TODO don't merge me - us separate fix in https://github.com/apache/kafka/pull/2734
```

Comment:
I need this to get tests to pass locally. Feel free to remove as I have a PR to change these running  separately.